app/components/m-header.hbs

```diff
@@ -1,87 +1,160 @@
-<div class="auk-navbar auk-navbar--bordered-bottom auk-navbar--no-pad auk-navbar--auto l-main-header"
-     id="c-main-nav">
-  <div class="auk-toolbar-complex auk-toolbar-complex--auto l-main-header__toolbar">
+<div class="auk-navbar auk-navbar--bordered-bottom auk-navbar--white auk-navbar--no-pad" id="c-main-nav">
+  <div class="auk-toolbar-complex">
     <div class="auk-toolbar-complex__left">
+      <div class="auk-toolbar-complex__item">
+        <WebComponents::AuBrand @logotype="Kaleidos"/>
+      </div>
       <div class="auk-toolbar-complex__item">
         <ul class="auk-tabs">
-          {{#if hasPublicationsEnabled}}
+          {{#if this.hasPublicationsEnabled}}
             <li class="auk-tabs__tab" data-test-m-header-publications>
-              {{#link-to
-                "publications"
-              }}
-                <span class="auk-tabs__tab-inner">{{t "publications"}}</span>
-              {{/link-to}}
+              <LinkTo @route="publications">
+                <span class="auk-tabs__tab-inner">
+                  <span class="auk-tabs__label">
+                    {{t "publications"}}
+                  </span>
+                </span>
+              </LinkTo>
             </li>
           {{/if}}
           <li class="auk-tabs__tab" data-test-m-header-agendas>
-            {{#link-to
-              "agendas.overview"
-              current-when="agendas agenda print-overviews.decisions print-overviews.press-agenda"
-            }}
-              <span class="auk-tabs__tab-inner">{{t "agendas"}}</span>
-            {{/link-to}}
+            <LinkTo @route="agendas.overview"
+                    @current-when="agendas agenda print-overviews.decisions print-overviews.press-agenda">
+              <span class="auk-tabs__tab-inner">
+                <span class="auk-tabs__label">
+                  {{t "agendas"}}
+                </span>
+              </span>
+            </LinkTo>
           </li>
```

Comment:
```
  <WebComponents::AuTab @route="agendas.overview" data-test-m-header-agendas>{{t "agendas"}}</WebComponents::AuTab>
  ```
  
  De current-when is nog niet iets dat je kan meegeven, maar simpel opgelost door die @current-when mee te geven naar au-tab.hbs lijn 13
```
{{else}}
  <li class="auk-tabs__tab" ...attributes>
    <LinkTo @route={{this.route}} @current-when={{this.current-when}}>
      <span class="auk-tabs__tab-inner">
        <WebComponents::AuTab::Innerlayout @layout={{this.layout}} @icon={{this.icon}} @counter={{this.counter}}>
          {{yield}}
        </WebComponents::AuTab::Innerlayout>
      </span>
    </LinkTo>
  </li>
{{/if}}
```